web/src/connectors/tasks/create-task/create-task.tsx

```diff
@@ -73,6 +74,7 @@ export const CreateTask: FC<ICreateNewTaskProps> = ({
                         <Text>{`Task is created with id=${response.id}`}</Text>
                     </>
                 );
+                svc.uuiModals.closeAll();
```

Comment:
I propose to close the modal outside "create-task.tsx"

Reply:
Logic changed